src/data/rds/2014-10-31/waiters-2.json

```diff
@@ -85,6 +85,91 @@
           "argument": "DBInstances[].DBInstanceStatus"
         }
       ]
+    },
+    "DBSnapshotAvailable": {
+      "delay": 30,
+      "operation": "DescribeDBSnapshots",
+      "maxAttempts": 60,
+      "acceptors": [
+        {
+          "expected": "available",
+          "matcher": "pathAll",
+          "state": "success",
+          "argument": "DBSnapshots[].Status"
+        },
+        {
+          "expected": "deleted",
+          "matcher": "pathAny",
+          "state": "failure",
+          "argument": "DBSnapshots[].Status"
+        },
+        {
+          "expected": "deleting",
+          "matcher": "pathAny",
+          "state": "failure",
+          "argument": "DBSnapshots[].Status"
+        },
+        {
+          "expected": "failed",
+          "matcher": "pathAny",
+          "state": "failure",
+          "argument": "DBSnapshots[].Status"
+        },
+        {
+          "expected": "incompatible-restore",
+          "matcher": "pathAny",
+          "state": "failure",
+          "argument": "DBSnapshots[].Status"
+        },
+        {
+          "expected": "incompatible-parameters",
+          "matcher": "pathAny",
+          "state": "failure",
+          "argument": "DBSnapshots[].Status"
+        }
+      ]
+    },
+    "DBSnapshotDeleted": {
+      "delay": 30,
+      "operation": "DescribeDBSnapshots",
+      "maxAttempts": 60,
+      "acceptors": [
+        {
+          "expected": "deleted",
+          "matcher": "pathAll",
+          "state": "success",
+          "argument": "DBSnapshots[].Status"
+        },
+        {
+          "expected": "DBInstanceNotFound",
```

Comment:
This should be `'DBSnapshotNotFound'`, as [specified here.](http://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_DescribeDBSnapshots.html#API_DescribeDBSnapshots_Errors) The waiters-2.json.php compile will also need to be redone.